### PR TITLE
[tests-only]Add tests for pagination in spaces

### PR DESCRIPTION
**Pull request by SwikritiT**
## Related Issue

- Part of: https://github.com/owncloud/ocis/issues/5013

## How Has This Been Tested?

- locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oCIS</strong> https://drone.owncloud.com/owncloud/web/35703/12/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/35703/tracing/pagination-alice-2023-5-16-05-21-25.zip 


</p></details>
:boom: The e2e-ocis tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oC10</strong> https://drone.owncloud.com/owncloud/web/35707/11/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/35707/tracing/pagination-alice-2023-5-16-07-46-45.zip 


</p></details>
:boom: The e2e-oc10 tests pipeline failed. The build has been cancelled.

File: tests/e2e/cucumber/steps/ui/resources.ts [hunk 1]
```diff
@@ -562,17 +562,18 @@ Then(
...
-  '{string} should see {int} resources in the personal space files view',
+  '{string} should see {int} resources in the personal/project space files view',
   async function (this: World, stepUser: string, expectedNumberOfResources: number) {
     const { page } = this.actorsEnvironment.getActor({ key: stepUser })
     const resourceObject = new objects.applicationFiles.Resource({ page })
-    const actualNumberOfResources = await resourceObject.getNumberOfResourcesInThePage()
-    await expect(actualNumberOfResources).toBe(expectedNumberOfResources)
+    await resourceObject.expectNumberOfResourcesInThePageToBe({
+      numberOfResources: expectedNumberOfResources
+    })
```
Comment (saw-jan) [inline tests/e2e/cucumber/steps/ui/resources.ts:571]: we should only get the required information from objects and assert here in step def
```suggestion
    const actualNumberOfResources = await resourceObject.getNumberOfResourcesInThePage()
    await expect(actualNumberOfResources).toBe(expectedNumberOfResources)
```

**Review by saw-jan — COMMENTED** [review]
LGTM

**Review by saw-jan — APPROVED** [review]

**Review by ScharfViktor — APPROVED** [review]

**Pull request merged**